CHANGELOG.md

```diff
@@ -103,6 +103,7 @@
     * _The App Store no longer accepts bitcode submissions from Xcode 14_
     * This version of PPRiskMagnes drops support for Xcode 12 and requires Swift 5.5+
       * [As of April 25, 2022 Apple requires all apps to be submitted with Xcode 13+](https://developer.apple.com/news/upcoming-requirements/?id=04252022a)
+    * This version of the PPRiskMagnes framework is dynamic. This reverts a breaking change that was introduced in minor version 5.8.0 (See GitHub issue #920).
```

Comment:
Should this go into the `5.x` branch? Since we are updating `master` what's on the `5.x` branch every so often that would get us coverage in both branches.

Reply:
Yes, great catch!

Reply:
Ok. I made a new PR - https://github.com/braintree/braintree_ios/pull/928.

I also accidentally committed directly to `5.x` 🤦, but just reverted it.

Reply:
🙈